src/app/projects/states/dashboard/directives/task-dashboard/directives/task-status-card/task-status-card.coffee

```diff
@@ -26,6 +26,7 @@ angular.module('doubtfire.projects.states.dashboard.directives.task-dashboard.di
 
     extendAndSubmit = (trigger) ->
```

Comment:
This should go now - this will loop infinitely now as the extensions must be granted to change the dates.

Reply:
Can delete `extendAndSubmit`

Reply:
Removed.